bundles/AdminBundle/Controller/Admin/UserController.php

```diff
@@ -1023,6 +1024,27 @@ public function onKernelResponse(FilterResponseEvent $event)
         // nothing to do
     }
 
+    /**
+     * @param Request $request
```

Comment:
seems that the commit is in the wrong branch?

Reply:
thx! you are absolutely right